src/bin/fullnode.rs

```diff
@@ -119,14 +124,14 @@ fn main() {
     }
 
     let exit = Arc::new(AtomicBool::new(false));
-    let threads = if matches.opt_present("v") {
-        let path = matches.opt_str("v").unwrap();
+    let threads = if matches.opt_present("t") {
+        let testnet = matches.opt_str("t").unwrap();
         eprintln!(
-            "starting validator... {} using {}",
-            repl_data.requests_addr, path
+            "starting validator... {} connecting to {}",
+            repl_data.requests_addr, testnet
         );
-        let file = File::open(path.clone()).expect(&format!("file not found: {}", path));
-        let leader = serde_json::from_reader(file).expect("parse");
+        let taddr = testnet.parse().unwrap();
+        let entry = ReplicatedData::new_entry_point(taddr);
```

Comment:
The term `entry` is already actively used throughout the codebase. Can you find another?